app/services/bucket_service.rb

```diff
@@ -8,9 +9,11 @@ def self.send_bucket_created_emails(bucket: )
   end
 
   def self.send_bucket_live_emails(bucket: )
-    memberships = bucket.group.memberships.active.reject { |membership| membership.member == bucket.user }
-    memberships.each do |membership|
-      member = membership.member
+    group = bucket.group
+    members = bucket.participants(exclude_author: true, subscribed: true)
+    members.each do |member|
```

Comment:
That would be ideal, but the archived attribute is bound to the membership, not the member